### ISSUE TITLE
Allow signed-in users to submit feedback anonymously

### ISSUE THREAD

**Issue by willdurand** [mozilla/addons#14706]
Signed-in should be able to report feedback anonymously. In order to do that we'll need to change the form but also avoid forwarding the credentials when we POST the data to the API.

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/ADDFRNT-258)
